packages/mui-material/src/ButtonBase/ButtonBase.js

```diff
@@ -47,24 +47,28 @@ export const ButtonBaseRoot = styled('button', {
   borderRadius: 0,
   padding: 0, // Remove the padding in Firefox
   cursor: 'pointer',
-  userSelect: 'none',
   verticalAlign: 'middle',
   MozAppearance: 'none', // Reset
   WebkitAppearance: 'none', // Reset
   textDecoration: 'none',
   // So we take precedent over the style of a native <a /> element.
   color: 'inherit',
+  '&:active': {
+    userSelect: 'none',
+  },
   '&::-moz-focus-inner': {
     borderStyle: 'none', // Remove Firefox dotted outline.
   },
   [`&.${buttonBaseClasses.disabled}`]: {
-    pointerEvents: 'none', // Disable link interactions
     cursor: 'default',
+    ...(!ownerState.isButton && {
```

Comment:
This breaks two things:

```suggestion
```

1. Hover style:

https://github.com/mui/material-ui/assets/3165635/31488ffc-e37d-4838-b7d3-682bff23fd66

2. Safari disabled button tooltip display

https://deploy-preview-38158--material-ui.netlify.app/material-ui/react-tooltip/#disabled-elements

Reply:
I removed it, the use of this was so disabled buttons text won't be clickable by double-click, but adding it seems to open up other issues I don't see worth opening.